js/apps/admin-ui/src/authentication/components/EditFlow.tsx

```diff
@@ -82,7 +82,7 @@ export const EditFlow = ({ execution, onRowChange }: EditFlowProps) => {
             <FormProvider {...form}>
               <TextControl
                 name="name"
-                label={t("name")}
+                label={t("displayName")}
```

Comment:
@agagancarczyk I'm not quite sure this fixes the issue, I had to change the value of the `name` attribute to fix it, while you are changing the value of the `label` attribute here.

Cf. with https://github.com/keycloak/keycloak/pull/30948/files